cc/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Is build the convention? I tend to prefer build.sh so people know it's a shell script. However, I'm not super fussed.

Reply:
I'm happy to name it whatever you'd like. My personally style is to never use extensions with executables. Callers shouldn't know or care, and we should be free to change, the implementation language without changing the name. Would you like me to rename it `build.sh`?